test/app-tests/loading.js

```diff
@@ -271,9 +245,30 @@ describe('loading:', function () {
             localStorage.setItem("mx_is_url", "http://localhost" );
             localStorage.setItem("mx_access_token", "access_token");
             localStorage.setItem("mx_user_id", "@me:localhost");
+            localStorage.setItem("mx_last_room_id", "!last_room:id");
```

Comment:
I'm not sure I like the fact that these tests rely on each other... Could this be moved to the start of the next test?